key,default_value
userAgent,"Boto3/1.7.4 Python/3.7.2+ Linux/4.4.0-01985-Microsoft Botocore/1.10.4"
eventID,"e29792df-1138-4825-9f50-4e26c3da558b"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: snowball:GetSnowballUsage"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"2.231.90.242"
eventName,"GetSnowballUsage"
eventSource,"snowball.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"c1ff0225-411e-11e9-a67a-01701eca06ea"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2019-03-07T21:19:57Z"
